### ISSUE TITLE
Links and web accessibility

### ISSUE THREAD

**Issue by christinemurr** [swcarpentry/r-novice-gapminder#734]
The links on the [Setup page](http://swcarpentry.github.io/r-novice-gapminder/setup.html) could be improved to meet web accessibility standards. People using screen readers may navigate a page by links, so the link descriptions should be meaningful, and "here" should be avoided. (See [WebAIM: Links and Hypertext](https://webaim.org/techniques/hypertext/#screen_readers).)

For example, the line "The latest version of R can be downloaded here" could we rewritten as: Download [the latest version of R](https://cran.r-project.org/mirrors.html).

**Comment by jcoliver** [comment]
Good point, @christinemurr . Want to submit a PR to address this?

**Comment by christinemurr** [comment]
I'm working on it, but...the CONTRIBUTING.md file says "Please only include
.Rmd files in pull requests," but I only see setup.md, no setup.Rmd. Am I
missing something, or is it okay to commit changes to setup.md?

**Comment by jcoliver** [comment]
Ah, sorry for the confusion. The information in the contributing document is referring to episodes, but since setup.md is not an episode, you should edit setup.md directly.

**Comment by lhami** [comment]
Looks like this was resolved in ecd0d398d71a87ca6f235814ff88e9edf2f03b3d
I think this should be marked as resolved.

**Comment by jcoliver** [comment]
Thanks, @lhami!  Addressed in #736.
